pills/01-why-you-should-give-it-try.xml

```diff
@@ -10,50 +10,50 @@
     <title>Introduction</title>
 
     <para>
-        Welcome to the first post of the <link
-        xlink:href="https://nixos.org/nix">Nix</link> in pills series.
-        Nix is a purely functional package manager and deployment
-        system for POSIX. There's a lot of documentation that
-        describes what Nix, <link
-        xlink:href="https://nixos.org/nixos">NixOS</link> and related
-        projects are.
+      Welcome to the first post of the "<link
+      xlink:href="https://nixos.org/nix">Nix</link> in pills" series.
+      Nix is a purely functional package manager and deployment
+      system for POSIX.
     </para>
 
     <para>
-      The purpose of this post is to convince you to give Nix a try.
+      There's a lot of documentation that describes what Nix, <link
+      xlink:href="https://nixos.org/nixos">NixOS</link> and related
+      projects are.
+      But the purpose of this post is to convince you to give Nix a try.
       Installing NixOS is not required, but sometimes I may refer to
       NixOS as a real world example of Nix usage for building a whole
       operating system.
     </para>
   </section>
 
   <section>
-    <title>Rationale of pills</title>
+    <title>Rationale for this series</title>
     <para>
-      The <link xlink:href="https://nixos.org/nix/manual/">Nix
-      manual</link>, <link
-      xlink:href="http://nixos.org/docs/papers.html">related
-      papers</link>, <link
-      xlink:href="http://www.infoq.com/articles/configuration-management-with-nix">articles</link>
-      and wiki are excellent in explaining how Nix/NixOS works, how
-      you can use it and the amount of cool things being done with it,
-      however sometimes you may feel some magic happening behind the
-      scenes hard to grasp in the beginning.
+      The <link xlink:href="https://nixos.org/nix/manual/">Nix</link>,
+      <link xlink:href="https://nixos.org/nixpkgs/manual/">Nixpkgs</link>, and
+      <link xlink:href="https://nixos.org/nixos/manual/">NixOS</link> manuals;
+      <link xlink:href="http://nixos.org/docs/papers.html">related papers</link>;
```

Comment:
This papers link is dead :(

Reply:
Hm, I'll remove it for now. I note that there's a `papers-in.html` at `nixos.org/docs` - possibly something has gone wrong with the site generation there?